cloudevents/extensions/loglevel.md

```diff
@@ -0,0 +1,167 @@
+# Log Level Extension
+
+## Abstract
+This extension defines attributes that MAY be included within a CloudEvent
+to describe the "level" or "importance" of an event in relation to other events
+produced by a unique event source.
+
+Often systems produce events in form of logs, and these types of events usually share
+a common concept of "log-level". This extension aims to provide a standard way for 
+describing this property in a language agnostic form. 
+
+Sharing a common way to describe importance of events allows for better monitoring 
+systems, tooling and general log consumption.
+
+In addition to defining the extension attributes, this document defines 
+the concept of [Log Level Bindings](#log-level-bindings) - a standard way to map 
+well-known log level names from different languages and frameworks 
+onto the `loglevelname` and `loglevelnum` values defined in this spec. 
+
+## Attributes
+
+All attributes of this extension are OPTIONAL. however, when using this 
+custom extension you MUST use at-least one of these OPTIONAL attributes. 
+
+### loglevelname 
+
+- Type: `String`
+- Description: Human readable representation of the log level
+- Constraints
+  - OPTIONAL
+  - if present, MUST be a non-empty string
+  - SHOULD be lowercase
+  - RECOMMENDED values include `emergency`, `alert`, `critical`, `error`, 
+    `warning`, `notice`, `info`, `debug`, and `verbose`, but  others MAY be used.
+  - SHOULD have a one-to-one relationship with a distinct `loglevelnum` in the scope of
+    the `source`.
+
+### loglevelnum 
+
+- Type: `Integer`
+- Description: A numerical representation of the level of importance of the event. 
+  Conforms to the severity ordering specified by 
+  [RFC5424](https://www.rfc-editor.org/rfc/rfc5424.html#section-6.2.1): severity of
+  all levels is assumed to be numerically ascending from most important to least
+  important.
+  This specification does not define "importance" or meaning of each value other than
```

Comment:
Add a blank line here, if it's intended to be a new paragraph? (And possibly before "Conforms" as well?)